google/cloud/bigtable/examples/README.md

```diff
@@ -3,25 +3,34 @@
 This directory contains minimal examples to get started quickly, and are
 referenced from the Doxygen landing page as "Hello World".
 
-## Compiling the Examples
+## Table of Contents
+
+- [Before you begin](#before-you-begin)  
+  - [Run the examples against the Cloud Bigtable Service](#run-the-examples-against-the-cloud-bigtable-service)
```

Comment:
Please look at the changes I requested in #745, in particular:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/pull/745/files#diff-0b42aa6f16cbd28c471e1b79b6315036R10